src/__test__/table-theme-colors.spec.js

```diff
@@ -0,0 +1,298 @@
+import tableThemeColors from '../table-theme-colors';
+
+describe('table-theme-colors', () => {
+  const spyQuerySelector = jest.fn();
+  const spyGetComputedStyle = jest.fn();
+  const spyGetStyle = jest.fn();
+  const setObjectAndTableBackgroundColorInTheme = (objectBackgroundColor, tableBackgroundColor) =>
+    jest.fn((base, path, attribute) => {
+      let backgroundColor;
+      if (attribute === 'backgroundColor') {
+        backgroundColor = objectBackgroundColor;
+      } else if (attribute === 'object.straightTable.backgroundColor') {
+        backgroundColor = tableBackgroundColor;
+      }
+      return backgroundColor;
+    });
+  const setqvPanelSheetAndqvInnerObject = jest.fn((selector) => selector);
```

Comment:
why do you need jest.fn? Isn't this just a function?

Reply:
yes, good point. Got it upated.